src/db-optm.cls.php

```diff
@@ -309,10 +309,15 @@ public function autoload_summary()
 	{
 		global $wpdb;
 
-		$summary = $wpdb->get_row("SELECT SUM(LENGTH(option_value)) AS autoload_size,COUNT(*) AS autload_entries FROM `$wpdb->options` WHERE autoload='yes'");
+		$autoloads = version_compare( get_bloginfo( 'version' ), '6.6.0', '>=' ) ? 
```

Comment:
Instead of version compare can we use `function_exists`?

Reply:
Thank you, changed